content/es/docs/tasks/debug-application-cluster/debug-init-containers.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Depurar Contenedores de Inicialización
+content_template: templates/task
+---
+
+{{% capture overview %}}
+
+Esta página muestra cómo investigar problemas relacionados con la ejecución
+de los contenedores de inicialización (init containers). Las líneas de comando de ejemplo de abajo 
+se refieren al Pod como `<pod-name>` y a los Init Containers como `<init-container-1>` e
+  `<init-container-2>` respectivamenet.
+
+{{% /capture %}}
+
+{{% capture prerequisites %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+* Deberías estar familizarizado con el concepto de [Init Containers](/docs/concepts/abstractions/init-containers/).
+* Deberías conocer la [Configuración de un Init Container](/docs/tasks/configure-pod-container/configure-pod-initialization/#creating-a-pod-that-has-an-init-container/).
+
+{{% /capture %}}
+
+{{% capture steps %}}
+
+## Comprobar el estado de los Init Containers
+
+Muestra el estado de tu pod:
+
+```shell
+kubectl get pod <pod-name>
+```
+
+Por ejemplo, un estado de `Init:1/2` indica que uno de los Init Containers
+se ha ejecutado satisfactoriamente:
+
+```
+NAME         READY     STATUS     RESTARTS   AGE
+<pod-name>   0/1       Init:1/2   0          7s
+```
+
+Echa un vistazo a [Comprender el estado de un Pod](#understanding-pod-status) para más ejemplos
+de valores de estado y sus significados.
+
+## Obtener detalles acerca de los Init Containers
+
+Ver más información detallada acerca de la ejecución de un Init Container:
+
+```shell
+kubectl describe pod <pod-name>
+```
+
+Por ejemplo, un Pod con dos Init Containers podría mostrar lo siguiente:
+
+```
+Init Containers:
+  <init-container-1>:
+    Container ID:    ...
+    ...
+    State:           Terminated
+      Reason:        Completed
+      Exit Code:     0
+      Started:       ...
+      Finished:      ...
+    Ready:           True
+    Restart Count:   0
+    ...
+  <init-container-2>:
+    Container ID:    ...
+    ...
+    State:           Waiting
+      Reason:        CrashLoopBackOff
+    Last State:      Terminated
+      Reason:        Error
+      Exit Code:     1
+      Started:       ...
+      Finished:      ...
+    Ready:           False
+    Restart Count:   3
+    ...
+```
+
+También peudes acceder al estado del Init Container de forma programática mediante
+la lectura del campo `status.initContainerStatuses` dentro del Pod Spec:
+
+
+```shell
+kubectl get pod nginx --template '{{.status.initContainerStatuses}}'
+```
+
+
+Este comando devolverá la misma información que arriba usando JSON plano.
```

Comment:
Texto original en ingles: This command will return the same information as above in raw JSON.

s/usando JSON plano/en formato JSON ?

Reply:
Cambiado